### DISCUSSION TITLE
Unable to insert an image in textblock

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#782] [General] [answered]
I try to insert an image under Opportunity in the Background textblock. I can't figure out why this doesn't work.
It seems to be somehow related to the format / size (?) of the image, because other images do work.

The image url  = https://gemeentelijkgeoberaad.pleio.nl/file/download/4732773c-792d-4365-92a1-9afa90236b6d/3dketen_inwinning.jpg

![afbeelding](https://user-images.githubusercontent.com/102361398/188603595-b890727a-3914-494d-b496-689225e18610.png)

**Comment by techsmyth** [comment, marked as answer]
Hi Gerlof, I just tried to open that image in a new browser and I am not able to access it: 

![image](https://user-images.githubusercontent.com/30729240/188695240-a1ccb627-bab1-4608-b1bf-49adbbd30505.png)

So I suspect that that link is not directly to the image but to server side loader that then loads the image. 

The ideal solution would I think be if you can directly load the image yourself onto Alkemio, right? So you would be able to somewhere upload an image and then refer to it in your content. We may need to prioritise that higher.

Cheers, Neil

**Reply by GerlofdeHaan** [reply]
Hi Neil, 
So that was the issue here. Until now I have inserted images in this way. But those imagese were all uploaded in the public sector of the server. I didn't check this properly for this particular image, so it was hidden behind an login. Changed the settings and it's working now.

thanks,
Gerlof

**Reply by techsmyth** [reply]
Hi Gerlof,

Nice that it is now working. 

Still we need to make uploading of images directly feasible, not everyone has the luxury of an alternative image hosting readily available...

Cheers, Neil
